src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx

```diff
@@ -13,101 +13,127 @@ redirects:
 ---
 
-import queriesnrqlChartTooltip from 'images/queries-nrql_screenshot-crop_chart-tooltip.png'
+import queriesnrqlChartType from 'images/queries-nrql_screenshot-full_chart-type.png'
 
-import queriesnrqlCustomChartTimestamp from 'images/queries-nrql_screenshot-crop_custom-chart-timestamp.png'
+import queriesnrqlCustomChartTimestamp from 'images/queries-nrql_screenshot-full_custom-chart-timestamp.png'
 
-import queriesnrqlCustomChartDateFormat from 'images/queries-nrql_screenshot-crop_custom-chart-date-format.png'
+import queriesnrqlCustomChartDateFormat from 'images/queries-nrql_screenshot-full_custom-chart-date-format.png'
 
-import queriesnrqlCustomChartDecimals from 'images/queries-nrql_screenshot-crop_custom-chart-decimals.png'
+import queriesnrqlCustomChartDecimals from 'images/queries-nrql_screenshot-full_custom-chart-decimals.png'
 
-import queriesnrqlCustomChartLegend from 'images/queries-nrql_screenshot-crop_custom-chart-legend.png'
+import queriesnrqlCustomChartLegend from 'images/queries-nrql_screenshot-full_custom-chart-legend.png'
 
-import queriesnrqlCustomChartOthers from 'images/queries-nrql_screenshot-crop_custom-chart-others.png'
+import queriesnrqlCustomChartOthers from 'images/queries-nrql_screenshot-full_custom-chart-others.png'
 
 Once you've created a chart, you can customize the appearance of it to best present the data. You can share a chart in different formats and add a chart to a new or existing dashboard.
 
 ## Change the appearance of your chart [#change-appearance]
 
 When you run your query in NRQL mode or view your chart while using the data explorer to specify data, New Relic analyzes your data and applies a [chart type](/docs/insights/use-insights-ui/manage-dashboards/insights-chart-types) that fits your data.
 
-For some queries, you'll have several options of chart types to choose from. To change chart type, use the **Chart type** menu to the right of the current chart. Each type in the list has a tooltip with information about using that type.
+You've several options of chart types to choose from. To change chart type, use the **Chart type** menu to the right of the current chart.
 
 <img
-  title="crop-cb-chart-menu-tooltip.png"
-  alt="crop-cb-chart-menu-tooltip.png"
-  src={queriesnrqlChartTooltip}
+  title="Chart type menu"
+  alt="Chart type menu"
+  src={queriesnrqlChartType}
 />
 
 <figcaption>
-  Example of the chart type menu, showing a tooltip.
+**[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Query your data**: Example of the chart type menu.
 </figcaption>
 
 ## Customize your charts [#customize-charts]
 
-While we try our best to optimize how we display your data, sometimes you may have other needs. Depending on the chart type, additional customization options are available.
+While we try our best to optimize how we display your data, sometimes you may have other needs. Depending on the chart type, there are available additional customization options.
```

Comment:
Suggest slightly modifying "there are available additional customization options" to "there are additional customization options available"

Reply:
Thank you! This one it's also much better ;-)